### ISSUE TITLE
Resize to top/bottom two-thirds

### ISSUE THREAD

**Issue by themadsens** [fifafu/BetterTouchTool#1994]
#### Feature request:
In "Trigger Predefined Actions" in section "Window Resize & Move" We have already:
  Resize to left/middle/right third and two-thirds

I would *really* like "Resize to top two-thirds" 
but in the name of symmetry there should probably be all the top/bottom third/two-thirds combinations

**Comment by Multiply** [comment]
This is also something I really need, or even better, let me select a percentage instead, so it doesn't have to be thirds.

My usecase is half of left screen for one window, top 70% of the right half of the screen for a second window, and the bottom 30% of the right half for a third window.

Sometimes I'd like to swap the size of the two windows on the right, so it's 30%-70% instead, and I can script my way out of that, if we allow for percentages.

**Comment by Multiply** [comment]
@fifafu I actually tried that when I first install the app, but for some reason whenever I created a new snap area, I didn't realize the little box that pops up after wasn't my selected area, and I got frustrated that every time I used active window as a template, it would only make a 'small' area.

I've just tried it again now, and it seems to work great. Guess I was just too impatient the first time around.

Thanks for the awesome tool, it works flawlessly.
